Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -47,18 +49,19 @@ public struct Conv1D<Scalar: TensorFlowFloatingPoint>: Layer {
     ///   - dilation: The dilation factor for the temporal dimension.
     public init(
         filter: Tensor<Scalar>,
-        bias: Tensor<Scalar>,
+        bias: Tensor<Scalar>?,
```

Comment:
How about adding a default argument value  `nil` to all `bias` arguments?
```suggestion
        bias: Tensor<Scalar>? = nil,
```

Reply:
Sure, I can add that on all inits.